src/main/java/com/minecolonies/coremod/tileentities/TileEntityWareHouse.java

```diff
@@ -219,7 +210,7 @@ private TileEntity searchMostEmptySlot()
     {
         int freeSlots = 0;
         TileEntity emptiestChest = null;
-        for (@NotNull final BlockPos pos : getBuilding().getAdditionalCountainers())
+        for (@NotNull final BlockPos pos : getBuilding().getContainers())
```

Comment:
think you need to exclude the warehouse hutblock here?

Reply:
why?

Reply:
The warehouse block is also a valid chest to drop off

Reply:
thought we are not using it, but ok

Reply:
We're using it, but it was broken in some cases. I fixed this in a recent PR